components/renderers/renderer.js

```diff
@@ -83,7 +80,9 @@ export class Renderer {
       // Line width of a drawn particle.
       particleWidth: 2,
       // Reduce particle count to this fraction (improves FPS).
-      particleReduction: 0.1
+      particleReduction: 0.1,
+      // Wind velocity at which particle intensity is maximum (m/s).
```

Comment:
It makes sense to have this option configurable, but you also need to add a line after 141 so this gets updated when a user calls start or update:

`this.config_.maxWindIntensity = config.maxWindIntensity || this.config_.maxWindIntensity;
`